# Adds a point to cartesian pose buffer
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
---
bool success
string msg
